documentation/develop/browser-extension-development-tools.md

```diff
@@ -66,7 +66,7 @@ Create a scaffold for your browser extensions. Provide the tool with extension n
 
 ![Extension Scaffolding Builder]({% asset "documentation/develop/Extension_scaffolding_builder.png" @optim @path %})
 
-To get started, [visit the tool online](http://webextensions.tech/) or [install it locally from npm](https://www.npmjs.com/package/create-web-ext).
+To get started, [visit the tool online](http://webextensions.in/) or [install it locally from npm](https://www.npmjs.com/package/create-web-ext).
```

Comment:
Can you please also change it to `https` everywhere for this domain too

Reply:
Okay, sure!